src/js/ui.js

```diff
@@ -0,0 +1,70 @@
+import { removeAllChildNodes } from "./utils.js";
+
+class Ui {
+  #lottoList;
+  #purchasedCount;
+  #purchasedLottos;
+  #checkWinningNumber;
+
+  constructor() {
+    this.#purchasedLottos = document.querySelector("#purchased-lottos");
+    this.#lottoList = document.querySelector("#lotto-list");
+    this.#checkWinningNumber = document.querySelector("#check-winning-number");
+    this.#purchasedCount = document.querySelector(".purchased-count");
+  }
+
+  displayLottoDetail(checked) {
+    this.#lottoList.querySelectorAll(".lotto-detail").forEach(($detail) => {
+      $detail.style.display = checked ? "inline-block" : "none";
+    });
+  }
+
+  onToggleReadMore(checked) {
+    checked
+      ? this.#lottoList.classList.add("flex-col")
+      : this.#lottoList.classList.remove("flex-col");
+
+    this.displayLottoDetail(checked);
+  }
+
+  createListItem(lotto) {
+    const $newListItem = document.createElement("li");
+    $newListItem.className = "d-flex mx-1 text-4xl";
+
+    const $listOfLottoNumber = document.createElement("span");
+    $listOfLottoNumber.className = "lotto-detail ml-1";
+    $listOfLottoNumber.style.display = "none";
+    $listOfLottoNumber.innerText = lotto.join(", ");
+
+    const $listOfLottoIcon = document.createElement("span");
+    $listOfLottoIcon.className = "lotto-icon";
+    $listOfLottoIcon.innerText = "🎟";
+
+    $newListItem.appendChild($listOfLottoIcon);
+    $newListItem.appendChild($listOfLottoNumber);
+
+    return $newListItem;
+  }
+
+  renderLottoElements(lottos) {
+    removeAllChildNodes(this.#lottoList);
+
+    lottos.forEach((lotto) => {
+      const createdListItem = this.createListItem(lotto);
```

Comment:
```suggestion
      const $lottoElmenet = this.getLottoElement(lotto);
```
element를 할당할 경우에 $를 변수명에 prefix를 붙혀주고 있으니 여기서도 통일해보면 어떨까요?
추가로 create도 반환한다는걸 인지할수도 있겠지만 더 정확하게 로또 Element를 반환한다고 명시하는게 좋아보였어요